avocado_vt/loader.py

```diff
@@ -43,7 +43,6 @@ def guest_listing(options):
               os.path.join(data_dir.get_data_dir(), 'images'))
     LOG.info("Available guests in config:")
     guest_name_parser = standalone_test.get_guest_name_parser(options)
-    guest_name_parser.only_filter('i440fx')
```

Comment:
You obviously mean this is done here:

https://github.com/avocado-framework/avocado-vt/blob/master/avocado_vt/options.py#L227

And here:

https://github.com/avocado-framework/avocado-vt/blob/master/virttest/defaults.py#L5

So, ACK.